features/exploredash/build.gradle

```diff
@@ -53,7 +53,6 @@ dependencies {
     // Core
     implementation 'androidx.core:core-ktx:1.6.0'
     implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation 'androidx.work:work-runtime-ktx:2.6.0'
```

Comment:
`ExploreSyncWorker` isn't in the explore-dash module, so this isn't needed atm. However, we might want to move the worker into the module since it would only be logical.